pkg/gui/controllers/helpers/confirmation_helper.go

```diff
@@ -382,3 +382,15 @@ func (self *ConfirmationHelper) IsPopupPanel(viewName string) bool {
 func (self *ConfirmationHelper) IsPopupPanelFocused() bool {
 	return self.IsPopupPanel(self.c.CurrentContext().GetViewName())
 }
+
+func (self *ConfirmationHelper) TooltipForMenuItem(menuItem *types.MenuItem) string {
+	tooltip := menuItem.Tooltip
+	if menuItem.DisabledReason != "" {
+		if tooltip != "" {
+			tooltip = tooltip + "\n\n"
+		}
+		tooltip = tooltip +
+			style.FgRed.Sprintf(self.c.Tr.DisabledMenuItemTooltip, menuItem.DisabledReason)
```

Comment:
```suggestion
			style.FgRed.Sprintf(self.c.Tr.DisabledMenuItemPrefix) + menuItem.DisabledReason
```
I'd only have `Disabled:` in red: that helps with finding the disabled message but doesn't obscure the message itself (as red is less readable than the default text colour as you say)

So I'd swap out `DisabledMenuItemTooltip` for `DisabledMenuItemPrefix` and set it to 'Disabled: '.

Reply:
Pushed a commit for this myself, YOLO